service/src/main/java/uk/gov/hmcts/reform/fpl/model/Respondent.java

```diff
@@ -29,6 +29,7 @@
     @NotNull(message = "You need to add details to respondents")
     private final RespondentParty party;
     private final String leadRespondentIndicator;
+    private String persistRepresentedBy;
```

Comment:
Probably this could be of type "YesNo"?

Reply:
CCD still treats fieldType 'YesOrNo' value as a string so unfortunately above still needs be a string.

Reply:
Ah ok we could probably achieve that by having some serializers / desialiers but it's not in scope of this. Would you mind opening tech debt?

Reply:
So what could be done here is that the YesNo could have a `@JsonProperty` annotation added to it for "Yes" and "No" respectively, the issue is that this will then break some parts of the service as they work on the assumption that the return value from the enum being serialised is going to be upper case.